<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginPage</class>
 <widget class="QMainWindow" name="loginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>349</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>503</width>
    <height>349</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>503</width>
    <height>349</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="emailText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>53</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEntry">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>93</y>
      <width>390</width>
      <height>21</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEntry">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>178</y>
      <width>390</width>
      <height>20</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>242</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancleButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>242</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
